### DISCUSSION TITLE
Some helper shortcuts to quickly switch to now playing tab (and back again)

### DISCUSSION THREAD

**Discussion by flightmansam** [minbrowser/min#2061] [Show and tell]
https://github.com/flightmansam/min/tree/now-playing-shortcut

I have added something that has been itching me for a bit. I am often listing to music in a separate task dedicated to random stuff in order to keep things tidy from my project task. In this little experiment I add a new bang `!playing` that will display all of the tabs across your browser that are playing audio (could be more than one if you're a bit funky). I also added some keyboard shortcuts: `ctrl+mod+p` will instantly switch to the most recent task that is playing audio. I also added `ctrl+mod+[` which will switch to your most previous location on min. 

Use case:
I am working on my garden designs project in its own task while listening to Mort Garson's [Plantasia](https://music.youtube.com/playlist?list=OLAK5uy_khfcOEuin5PG2Z70byPFVH4nyEP_R8Bqc) in my general task. But then I want to change to something else, maybe some funk. So I hit `ctrl+mod+p` to take me to my Youtube Music tab and find a new album. Then I hit `ctrl+mod+[` and I am back working on garden designs. 

Nifty!

**Comment by PalmerAL** [comment]
Good idea! PR?

Actually, this seems like an ideal use case for having pinned tabs that stay in the tab bar regardless of which task you're in. Maybe we should do that...
